recipes/libnice/all/test_package/conanfile.py

```diff
@@ -0,0 +1,28 @@
+import os
+from conan import ConanFile
+from conan.tools.meson import Meson
+from conan.tools.layout import basic_layout
+from conan.tools.build import cross_building
+
+
+class LibniceTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "PkgConfigDeps", "MesonToolchain"
```

Comment:
```suggestion
    generators = "PkgConfigDeps", "MesonToolchain", "VirtualRunEnv"
```
To use conanrun environment, you need first, generate it.